test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -81,3 +82,13 @@ def test_tensorflow_telemetry_gpu(tensorflow_training, ec2_connection, gpu_only)
 @pytest.mark.parametrize("ec2_instance_type", ["c5.4xlarge"], indirect=True)
 def test_tensorflow_telemetry_cpu(tensorflow_training, ec2_connection, cpu_only):
     execute_ec2_training_test(ec2_connection, tensorflow_training, TF_TELEMETRY_CMD)
+
+# Testing Tensorboard with profiling
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_GPU_INSTANCE_TYPE, indirect=True)
+def test_tensorflow_tensorboard(tensorflow_training, ec2_connection, gpu_only):
```

Comment:
Could you set 2 test with different names?  e.g. "test_tensorflow_tensorboard-gpu"

Reply:
Made the changes.